File: Java/Iniciante/1038 - Lanche.java
/*
* Autor: Pierre Vieira
* Data de submissão: 29/02/2020 20:59:09
*/
import java.util.Scanner;
public class Main {


    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int codigo, quantidade;
        float preco = 0;
        codigo = teclado.nextInt();
        quantidade = teclado.nextInt();
        switch (codigo){
            case 1:
                preco += 4;
                break;
            case 2:
                preco += 4.5;
                break;
            case 3:
                preco += 5;
                break;
            case 4:
                preco += 2;
                break;
            case 5:
                preco += 1.5;

        }
        System.out.printf("Total: R$ %.2f\n", preco*quantidade);
    }

}

File: C#/Iniciante/1038 - Lanche.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 09:02:20
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int codigo, quantidade;
            double[] precos = {4.00, 4.50, 5.00, 2.00, 1.50};
            string[] linha = Console.ReadLine().Split(' ');
            codigo = int.Parse(linha[0]);
            quantidade = int.Parse(linha[1]);
            Console.WriteLine($"Total: R$ {(quantidade*precos[codigo - 1]).ToString("F2", CultureInfo.InvariantCulture)}");
        }
    }
}
